.github/workflows/ci.yml

```diff
@@ -0,0 +1,147 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  pull_request: {}
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Package version (e.g. v0.1.0)
+        required: false
+
+env:
+  # Common versions
+  GO_VERSION: '1.21.3'
+  GOLANGCI_VERSION: 'v1.54.2'
+  DOCKER_BUILDX_VERSION: 'v0.11.2'
+  XP_CHANNEL: master   # TODO(negz): Pin to stable once v1.14 is released.
+  XP_VERSION: current  # TODO(negz): Pin to a version once v1.14 is released.
+
+  # Common users. We can't run a step 'if secrets.USR != ""' but we can run
```

Comment:
i'm not familiar with `secrets.USR` and `env.USR` are defined and how they relate to `XPKG_ACCESS_ID` - are they common github vars? any extra context that should be added here?

Reply:
It was meant as an example value to demonstrate the different between secrets.WHATEVER and env.WHATEVER. I deleted this comment.